audio/Makefile

```diff
@@ -75,7 +77,7 @@ $(BIN): $(OBJS)
 
 depend: .depend
 
-clean:
+clean: .clean
```

Comment:
I think it's better to extend the target with a double colon.
The top-level audio/Makefile can have the common target and other Make.defs include audio/Makefile and extend `clean::` with their specific operations.